pyconcz_2016/speakers/migrations/0002_talk.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.3 on 2016-09-17 02:34
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('speakers', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Talk',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('title', models.CharField(max_length=200)),
+                ('description', models.CharField(blank=True, max_length=255)),
+                ('speaker', models.OneToOneField(on_delete=django.db.models.deletion.CASCADE, to='speakers.Speaker')),
```

Comment:
Uh oh, that raises some questions then (I was hoping there wouldn't be any cases like that this year and we could just ignore the possibility for now :D)

Possible solutions I can think of:

- Change this whole page to be a list of talks instead of list of speakers. This would be the exact opposite of what @beatrix-bodo told me should be done, but might make more sense.
- When Ms. Foo and Mr. Bar talk about Baz together, we could display both

   >Ms. Foo: Baz (with Mr. Bar)

   and

   >Mr. Bar: Baz (with Ms. Foo)

   as two separate entries on this page.

Which one should it be?